logger/logger.go

```diff
@@ -0,0 +1,59 @@
+package logger
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+	graylog "gopkg.in/gemnasium/logrus-graylog-hook.v2"
+)
+
+// ErrFailedToConfigureLog is the error returned when configuring failed for some reasons
+var ErrFailedToConfigureLog = errors.New("Failed to init log: failed to configure ")
```

Comment:
Please read this wiki page
https://github.com/golang/go/wiki/Errors#naming

Reply:
changed name and description

Reply:
fixed